store/auth_test.go

```diff
@@ -49,6 +49,15 @@ const mockStoreNeeds2fa = `
 }
 `
 
+const mockStore2faFailedHTTPCode = 403
+const mockStore2faFailedResponse = `
+{
+    "meessage": "The provided 2-factor key is not recognised.", 
```

Comment:
Good catch. And yes, message is there for completeness but we don't currently use it for anything